client/src/Components/CreateNewRoom/CreateNewRoom.jsx

```diff
@@ -4,15 +4,20 @@ import axios from 'axios';
 
 import VoteButton from 'Components/VoteButton/index.js';
 import CreateNewRoomStyles from './CreateNewRoomStyles';
+import { useRoomContext } from 'Contexts/RoomContext';
```

Comment:
Nie podoba mi się że to korzysta z room context. Jeżeli już chcemy wykorzystać kontekst do przechwytywania informacji to powinniśmy raczej stworzyć osobny context.  W tym konkretnym przypadku jednak zastanowiłbym się na wy renderowaniu snackbara z tego komponentu. Wtedy trzeba było część wizualną z ErrorBoxa wydzielić do nowego komponentu i użyć go w dwóch miejscach